src/aat/resources/features/F-026/S-589.td.json

```diff
@@ -33,75 +33,40 @@
 			"Transfer-Encoding": "[[ANY_STRING_NULLABLE]]"
 		},
 		"body": {
-			"columns": [
-				{
-					"label": "Last State Modified Date",
-					"order": 1,
-					"metadata": true,
-					"case_field_id": "[LAST_STATE_MODIFIED_DATE]",
-					"case_field_type": {
-						"id": "DateTime",
-						"type": "DateTime",
-						"min": null,
-						"max": null,
-						"regular_expression": null,
-						"fixed_list_items": [],
-						"complex_fields": [],
-						"collection_field_type": null
-					},
-					"display_context_parameter": null
-				},
-				{
-					"label": "`Text` field",
-					"order": 2,
-					"metadata": false,
-					"case_field_id": "TextField",
-					"case_field_type": {
-						"id": "Text",
-						"type": "Text",
-						"min": null,
-						"max": null,
-						"regular_expression": null,
-						"fixed_list_items": [],
-						"complex_fields": [],
-						"collection_field_type": null
-					},
-					"display_context_parameter": null
-				},
-				{
-					"label": "`Number` field",
-					"order": 3,
-					"metadata": false,
-					"case_field_id": "NumberField",
-					"case_field_type": {
-						"id": "Number",
-						"type": "Number",
-						"min": null,
-						"max": null,
-						"regular_expression": null,
-						"fixed_list_items": [],
-						"complex_fields": [],
-						"collection_field_type": null
-					},
-					"display_context_parameter": null
-				},
-				{
-					"label": "`YesOrNo` field",
-					"order": 4,
-					"metadata": false,
-					"case_field_id": "YesOrNoField",
-					"case_field_type": {
-						"id": "YesOrNo",
-						"type": "YesOrNo",
-						"min": null,
-						"max": null,
-						"regular_expression": null,
-						"fixed_list_items": [],
-						"complex_fields": [],
-						"collection_field_type": null
-					},
-					"display_context_parameter": null
-				}
+			"columns": [{
+        "_extends_": "F-026_ResponseData",
+        "metadata": true
+      },
+        {
+          "_extends_": "F-026_ResponseData",
+          "label": "`Text` field",
+          "order": 2,
+          "case_field_id": "TextField",
+          "case_field_type": {
+            "id": "Text",
+            "type": "Text"
+          }
+        },
+        {
+          "_extends_": "F-026_ResponseData",
+          "label": "`Number` field",
+          "order": 3,
+          "case_field_id": "NumberField",
+          "case_field_type": {
+            "id": "Number",
+            "type": "Number"
+          }
+        },
+        {
+          "_extends_": "F-026_ResponseData",
+          "label": "`YesOrNo` field",
+          "order": 4,
+          "case_field_id": "YesOrNoField",
+          "case_field_type": {
+            "id": "YesOrNo",
+            "type": "YesOrNo"
+          }
+        }
```

Comment:
there could be an additional layer or generalisation to use most details from their respective base objects.